src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -327,14 +326,49 @@ public static bool IsLinuxConsumptionOnAtlas(this IEnvironment environment)
                    string.IsNullOrEmpty(environment.GetEnvironmentVariable(LegionServiceHost));
         }
 
-        private static bool IsLinuxConsumptionOnLegion(this IEnvironment environment)
+        private static bool IsConsumptionOnLegion(this IEnvironment environment)
```

Comment:
Agree with the rename on this, since it returns true for both Linux Consumption and Flex Consumption SKUs running on Legion.

I think you should retain the comment I added on the method though in my original PR "Returns a value indicating whether the app is Linux Consumption running on Legion (either V1 or Flex)." which makes this clear